### ISSUE TITLE
Battle - wrog damage calculation for jihads

### ISSUE THREAD

**Issue by ccmaster001** [sirgwain/craig-stars#107]
![image](https://github.com/sirgwain/craig-stars/assets/170196074/01620454-a92d-48ef-8f06-88a874486866)

![image](https://github.com/sirgwain/craig-stars/assets/170196074/6875dfa8-21d3-4acd-bcc2-3d2e4d2d0fa1)

**Comment by sirgwain** [comment]
It doubled the damage because it's a capital ship missile and the target had no shields remaining. `85*2 = 170`

**Comment by ccmaster001** [comment]
Yes but the missile lot has at least 2 missiles so it should be 2*2*85 = 340
Also it no matteres if it is a capital ship or not. Missiles always do double damage if no shields are left.

**Comment by sirgwain** [comment]
Only remaining issue is double damage when shields gone for non capital ship missiles.

**Comment by sirgwain** [comment]
I don’t think regular torpedos do double damage when shields are gone. If that were the case, what is the point of capital ship missiles?

**Comment by ccmaster001** [comment]
All missles do double damage if no shields are out. Jihads are missles. The BB slot has 2 missles so damage is 2 × 2 ×85. What is not 170 damage for a single slot.
